nototools/generate_website_2_data.py

```diff
@@ -246,8 +246,7 @@ def get_family_id_to_regions(family_id_to_lang_scr_to_sample_key):
 
   family_id_to_regions = collections.defaultdict(set)
   warnings = set()
-  for (family_id, lang_scr_to_sample_key in
-       family_id_to_lang_scr_to_sample_key.iteritems()):
+  for family_id, lang_scr_to_sample_key in family_id_to_lang_scr_to_sample_key.iteritems():
```

Comment:
When I tried to run this with a wrapped line, it failed.  Thought I had run it with the wrapped line, but maybe I hadn't.  Anyway, I had to unwrap it to make it work.

Should the wrapped version work?